pkg/util/util.go

```diff
@@ -119,6 +118,59 @@ func GetBillingAccountFromSecret(kubeClient kubeclientpkg.Client, namespace, nam
 	return billingAccount, nil
 }
 
+// NewConfigMapCR creates a ConfigMap in specified namespace with default orgParentFolderID value
+func NewConfigMapCR(name, namespace, orgParentFolderID string) *corev1.ConfigMap {
```

Comment:
is a ConfigMap a CR? I don't think so, so maybe the method should be called differently

Reply:
CR is removed now.